drawBot/context/baseContext.py

```diff
@@ -904,7 +904,8 @@ def block(value, rng, stop):
         attributedSubstring = attributedString.attributedSubstringFromRange_(rng)
         # strip trailing returns
         if attributedSubstring.string()[-1] in ["\n", "\r"]:
-            attributedSubstring.deleteCharactersInRange_((rng.length - 1, 1))
+            rng.length -= 1
```

Comment:
an other option is to call `attributedSubstring.mutableCopy()` before `deleteCharactersInRange_`

Reply:
Same difference, pretty much...